src/tests/Microsoft.PowerFx.Core.Tests.Shared/ExpressionTestCases/JSON.txt

```diff
@@ -133,3 +133,60 @@ Error({Kind:ErrorKind.Div0})
 // Flattening nested tables
 >> JSON([[1,2,3],[4,5],[6]], JSONFormat.FlattenValueTables)
 "[[1,2,3],[4,5],[6]]"
+
+>> JSON(ParseJSON("{}"))
```

Comment:
>>> [](http://example.com/codeflow?start=0&length=2)

Can we call out an example that does not round-trip? 
And mention in comments // Does not roundtrip  because ... 

The test cases help us document the feature.

Reply:
C# tests give an example